### PR TITLE
Fix : Notebookbar Labels Decrease font size

### PR DESCRIPTION
**Pull request by codewithvk**
Signed-off-by: codewithvk <vivekpatel7202@gmail.com>
Change-Id: I151db6e80a446eefdee21b5536e00694b9734074

* Resolves: #2002 
* Target version: master 

### Summary
temp

### CODE REVIEW

**Comment by gokaysatir** [comment]
Hello, Thank you for the patch:)

@pedropintosilva i think your opinion is more important then mine here:)

**Review by pedropintosilva — CHANGES_REQUESTED** [review]
Thanks for your patch! But unfortunately this does not affect those labels (it only affects nested widgets , `ui-widget .ui-widget`). Could you please change it? See my comments.

also 0.5em is extremely small, please make sure you able to run Collabora Online and test it, let me know if there is something I can help with : )

File: loleaflet/css/jquery-ui-lightness.css [hunk 1]
```diff
@@ -881,7 +881,7 @@ button.ui-button::-moz-focus-inner {
 	font-size: 1.1em;
 }
 .ui-widget .ui-widget {
-	font-size: 1em;
+	font-size: 0.5em;
 }
```
Comment (pedropintosilva) [inline loleaflet/css/jquery-ui-lightness.css:885]: not the elements we are trying to target. Maybe they can then be removed (if they are not used anywhere else but for now and to make the things simple I would avoid changing this)

File: loleaflet/css/jquery-ui-lightness.css [hunk 2]
```diff
@@ -881,7 +881,7 @@ button.ui-button::-moz-focus-inner {
 	font-size: 1.1em;
```
Comment (pedropintosilva) [inline loleaflet/css/jquery-ui-lightness.css:881]: property of the intended element we are trying to target, maybe change it to 1em?

**Review by codewithvk — COMMENTED** [review]
Hey, I think this property not for our target, because it changes only hover text size.
temp
Can you please confirm me?

**Comment by pedropintosilva** [comment]
> Hey, I think this property not for our target, because it changes only hover text size.

Hm n0o, `.ui-widget, [data-title]:after` is what we want see:
image

**Review by pedropintosilva — APPROVED** [review]

**Comment by pedropintosilva** [comment]
Thanks! We are currently having troubles with tests but I think we can just merge this : ) :+1:

**Comment by codewithvk** [comment]
@pedropintosilva  Thanks for merge! This is my first contribution to CollaboraOnline project.
Looking forward more contribution :)

**Comment by pedropintosilva** [comment]
I'm glad I could merge your very first PR. We also have also many [easyhacks](https://github.com/CollaboraOnline/online/issues?q=is%3Aissue+is%3Aopen+label%3A%22Easy+Hack%22) and many other [issues](https://github.com/CollaboraOnline/online/issues) to play with : ) ping me by @ me or chatting in our IRC: irc://irc.freenode.net/#cool-dev

**Comment by pedropintosilva** [comment]
or on telegram if you prefer, all links listed in: https://collaboraonline.github.io/post/communicate/

**Comment by codewithvk** [comment]
hey @pedropintosilva ,
when I had to press join button telegram to show me this
temp

**Pull request merged**